docs/Simulation/index.rst

```diff
@@ -1,15 +1,20 @@
 Simulators
 =======================
 
-Chipyard provides support and integration for multiple simulation flows, for various user levels and requirements.
-In the majority of cases during a digital design development process, a simple software RTL simulation will do.
-When more advanced full-system evaluation is required, with long running workloads, FPGA-accelerated simulation will then become a preferable solution.
-The following pages provide detailed information about the simulation possibilities within the Chipyard framework.
+Chipyard supports two classes of simulation:
+
+#. Software RTL simulation using commercial or open-source (Verilator) RTL simulators 
+#. FPGA-accelerated full-system simulation using FireSim
+
```

Comment:
I would still advocate for including the example use-cases for SW vs. FPGA, because I have a hunch some people will see the O(100 MHz) and try to jump to FPGA, while not realizing that for 90% of use-cases (because many people will just test bare-metal assembly tests) SW sim will do fine.

Reply:
I'm not sure we should hand hold people that don't already know that. I'll make the costs more obvious though.

This was a bit of a knee jerk reaction to the amount of text i've seen reading the docs that explain things that either should be common knowledge (i.e., a new student could ask their non-berkeley colleague about it) or highly google-able. I think if were going to burn words on docs they should probably air on side of being more technical.

Reply:
We'll agree to disagree here :).
I've had conversations with at least two different group trying get FireSim setup just to run assembly tests (clearly, FireSim in unnecessary for that). You could argue that "this is common knowledge", or "they should know this", but since we are promoting FireSim (and I don't think anyone is currently promoting the benefits of SW RTL Sim in the outside world), I think people tend to gravitate to the product that was promoted, even if they don't need it.

Reply:
To be clear, i'm not saying necessarily that they should know it, but rather it's information readily available elsewhere and so have other means to learn it.

I worry if we have too much text people won't actually read the docs closely enough.